pepper-apis/README.md

```diff
@@ -6,14 +6,18 @@ You'll need a few things to get up and running for local development:
 
 * Java 11 - see the [docs here](docs/java-11.md)
 * Maven - for building the project
-* Python - for running various scripts if needed
+* Python - for running various scripts if needed (latest 2.x.x version)
 * Ruby - for rendering configuration files from templates
 * Google Cloud SDK - for running pubsub locally
 * Docker - for building images and optionally for running the test suite
 * Vault - for secrets management
 
 ### Setting up Google Cloud SDK
 
+Before installation of google components please make sure you have Python 2.x.x
+installed and environment variable CLOUDSDK_PYTHON is pointing to the python.exe
+(for example CLOUDSDK_PYTHON=c:\python\python.exe)
```

Comment:
This looks very Windows-specific. Again please double check cloud sdk docs and if setting up this env variable is needed, do explain that it is for Windows (I don't have it in my MacOS machine but gcloud runs fine for me).

Reply:
The Cloud SDK [docs](https://cloud.google.com/sdk/docs/install) explicitly state a version of Python 3 between 3.5 and 3.8 is recommended.

Reply:
Will recheck this on Mac. If it works, then we can decline this PR